cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -315,12 +304,9 @@ public void getAll(@NotNull Context ctx) {
                 ctx.status(HttpServletResponse.SC_OK);
                 ctx.result(results);
                 requestResultSize.update(results.length());
-                if (isLegacyVersion)
-                {
+                if (isLegacyVersion) {
```

Comment:
isn't this always false given the if statement above?

Reply:
Not necessarily. If the format parameter is specified, the statement above will be skipped and the version can still be either legacy or not.

Reply:
oh I see, it's one of those that uses the format parameter, not just the header